velox/docs/develop/unsaferow.rst

```diff
@@ -24,6 +24,9 @@ fixed-width columns (booleans, integers, floating point numbers) are stored
 directly. These values must fit within 8 bytes. Long decimal columns are not
 supported.
 
+UNKNOWN type (i.e NullType in Spark type system) is serialized as a fixed-width
```

Comment:
> with 1 byte.

Does this match Spark's implementation? I have an impression that all fixed-width types use 8 bytes.